packages/vue-instantsearch/src/components/CurrentRefinements.vue

```diff
@@ -33,13 +33,11 @@
                   <q v-if="refinement.attribute === 'query'">{{
                     refinement.label
                   }}</q>
-                  <template v-else>
-                    {{ refinement.label }}
-                  </template>
-                </span>
-                <button
+                  <template v-else>{{ refinement.label }}</template> </span
+                ><button
                   :class="suit('delete')"
-                  @click="item.refine(refinement)"
+                  type="button"
+                  @click.left.exact="item.refine(refinement)"
```

Comment:
This aligns with other flavours preventing refinement when a modifier key is pressed at the time of clicking on the button. Don't think it's a breaking change.

Reply:
just to double check before we commit this, does this transpile and work correctly in both vue 2 and vue 3?

Reply:
Tests pass (both `unit tests` and `vue v3`) and I just checked with Vue 3 [in a sandbox](https://codesandbox.io/s/suspicious-goldwasser-mlwms3?file=/src/App.vue) it works as expected.